doc/api/Renter.md

```diff
@@ -269,8 +281,15 @@ uploads a file to the network from the local filesystem.
 
 ###### Query String Parameters
 ```
+// The number of data pieces to use when erasure coding the file.
+datapieces // int
+
+// The number of parity pieces to use when erasure coding the file. Total
+// redundancy of the file is (datapieces+paritypieces)/datapieces.
+paritypieces // int
```

Comment:
this comment confused me lol. I think you just meant to say 'good job' though. At first I thought I had misformatted the comment, but I don't think there's a mistake here.